packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -7,15 +7,17 @@
         v-for="(route, key) in navigationRoutes"
         :key="key"
       >
-        <template #header="{ value }">
-          <va-sidebar-item :activeColor="activeColor" @keydown.stop>
-            <va-sidebar-item-content :class="{ 'va-sidebar-item--active': isRouteHasActiveChild(route) }">
-              <va-sidebar-item-title>
-                {{ $t(route.displayName) }}
-              </va-sidebar-item-title>
-              <va-icon v-if="route.children" :name="value ? 'expand_less' : 'expand_more'" />
-            </va-sidebar-item-content>
-          </va-sidebar-item>
+        <template #header="{ value, hasKeyboardFocus }">
+          <div
+            class="sidebar__collapse-custom-header"
+            :class="{
+              'sidebar__collapse-custom-header--active': isRouteHasActiveChild(route),
+              'sidebar__collapse-custom-header--keyboard-focused': hasKeyboardFocus
+            }"
+          >
+            {{ $t(route.displayName) }}
+            <va-icon v-if="route.children" :name="value ? 'expand_less' : 'expand_more'" />
```

Comment:
```
v-if="route.children"
```
Can you check if this check is relevant? IMO if route doesn't have children - it shouldn't be collapse.